### ISSUE TITLE
Changes on host don't sync by unison

### ISSUE THREAD

**Issue by yppon** [EugenMayer/docker-sync#300]
Docker-sync seems to be start correctly, but changes on host don't sync.
```
docker-sync start
       note:  You can also run docker-sync in the background with --daemon
          ok  Starting unison
          ok  web-sync container still running, restarting unison in container
          ok  Synced /Library/WebServer/Documents/Project/
     success  Unison server started
          ok  Synced /Library/WebServer/Documents/Project/
Contacting server...
Connected [//71e17fd42233//app -> //Macintosh-2.local//Library/WebServer/Documents/Project]
Looking for changes
  Waiting for changes from server
Reconciling changes
Nothing to do: replicas have not changed since last sync.
```
However, when I make the change directly under the root (e.g. touch test.txt),  other changes also sync correctly.

My `docker-sync.yml` is below.
```
version: '2'

syncs:
  web-sync:
    src: './'
    dest: '/app'
    sync_host_ip: '127.0.0.1'
    sync_host_port: 10871
    sync_strategy: 'unison'
```

What the cause of this issue ... ?

**Comment by EugenMayer** [comment]
Please check the watchdog issue, we will release an new version soon fixing this out of the box

**Comment by chris-aeviator** [comment]
@EugenMayer 

which one is the watchdog issue? I'm experiencing the same issue, but remember it was working fine some days ago. also if I checkout a different branch I can see files beeing synced.

**Comment by EugenMayer** [comment]
pip install watchdog

**Comment by EugenMayer** [comment]
The issues / fix in charge is https://github.com/EugenMayer/docker-sync/pull/295

**Comment by EugenMayer** [comment]
will be resolved with 0.3.0
